src/pages/About/MissionAndPurpose/MissionAndPurpose.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+// import PropTypes from 'prop-types'
+import Container from '../../../components/Container/Container.js'
+
+import './MissionAndPurpose.scss';
+
+export const AboutMissionTemplate = ({
+    section_uno,
+    section_two,
+    section_three,
+    }) => (
+        <div className="about-mission-purpose--content" style= {{textAlign:"left"}}>
```

Comment:
```suggestion
        <div className="about-mission-purpose">
```
about-mission-purpose will do just fine and please keep the styling in the css file